pkg/multicluster/watcher.go

```diff
@@ -28,3 +28,10 @@ type ClusterSet interface {
 	// List the clusters (sorted) currently known to the set
 	ListClusters() []string
 }
+
+// the multicluster Interface provides a handle to interacting with multiple clusters.
+// the multicluster watcher implements this interface.
+type Interface interface {
```

Comment:
Is `ClusterSet` specifically missing from this interface? and If so do we still wanna check that the `clusterWatcher` implements it?

Reply:
`Watcher` already includes `ClusterSet` (so it's actually a compile error to include them both here)